olcut-core/src/main/java/com/oracle/labs/mlrg/olcut/util/SubprocessConnection.java

```diff
@@ -45,16 +49,21 @@
 /**
  * A connection to a sub-process that can be communicated with over stdio. The subprocess
  * starts implicitly on activity or can be started explicitly. An idle timeout can also
- * be used to shut the process down when not in use. Subprocesses should be implemented to take
- * commands or opcodes and return output followed by an empty line as a response. They should also
- * watch for a "SHUTDOWN" command (the word "SHUTDOWN" on a line by itself) for terminating
- * the subprocess cleanly.
+ * be used to shut the process down when not in use.
+ *
+ * When a subprocess starts up, it should output the string "Ready" on a line by itself
```

Comment:
That's not what it actually checks for though, it wants `output.toLowerCase(Locale.ROOT).equals("ready")`, so maybe this should be lower case? Or specify that it's case insensitive?